aten/src/ATen/native/sparse/SparseUtils.h

```diff
@@ -42,20 +42,29 @@ inline bool _check_device(ArrayRef<Tensor> ts) {
   return true;
 }
 
-inline void _raw_resize_sparse(const SparseTensor& self, int64_t sparseDims, int64_t denseDims, IntList size) {
+// TODO: remove this function when is USE_TH_SIZE_ZERO_DIM enabled by default
+inline void _raw_resize_sparse_legacy(const SparseTensor& self, int64_t sparseDims, int64_t denseDims, IntList size) {
+#ifndef USE_TH_SIZE_ZERO_DIM
   _get_sparse_impl(self)->raw_resize_(sparseDims, denseDims, size);
+#endif
 }
 
+#ifndef USE_TH_SIZE_ZERO_DIM
```

Comment:
this seems fine as a wrapper, but also fine to get rid of if you prefer.

Reply:
It's just generated a bunch more churn in the diff to get rid of this when it's not necessary.

Reply:
Removing the `_alias_into_sparse` in the existing code probably won't save us a lot because it will be replaced with `set_indices_and_values_unsafe` and we still need to clean them up when the flag is enabled by default. We should probably just leave the wrapper as is right now.